app/src/ui/app.tsx

```diff
@@ -325,7 +325,9 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.reportStats()
     setInterval(() => this.props.dispatcher.reportStats(), SendStatsInterval)
 
-    this.props.dispatcher.installGlobalLFSFilters(false)
+    const selectedRepository = this.props.appStore.getSelectedRepository()
```

Comment:
I'm not a fan of this I don't think...this is global loading stuff and I'm not sure I know what we would want to do about this during remote execution.